redbeat/schedulers.py

```diff
@@ -266,7 +266,7 @@ def is_due(self):
             return False, 5.0  # 5 second delay for re-enable.
 
         return self.schedule.is_due(self.last_run_at or
-                                    datetime(MINYEAR, 1, 1, tzinfo=self.schedule.tz))
+                                    datetime(1970, 1, 2, tzinfo=self.schedule.tz))
```

Comment:
Literal constants always set off alarms in my head but I don't see a good way to eliminate them in this case.